packages/patterns-react/src/patterns/SimpleLongForm/SimpleLongForm.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import { settings } from 'carbon-components';
+import { featureFlag } from '@carbon/ibmdotcom-utilities';
+import { SIMPLELONGFORM } from '../../internal/FeatureFlags';
+import JumpLink from './JumpLink';
+import CardLink from './CardLink';
+import SimpleLink from './SimpleLink';
+
+const { prefix } = settings;
+
+/**
+ * Simple long form pattern
+ *
+ * @param {object} props props object
+ * @param {string} props.title simple long form title
+ * @param {string} props.copy simple long form  short copy to support the title
+ * @param {string} props.variation variation of the simple long form standard, standard with jump link and standard with card link
+ * @param {object} props.link link object which includes url, link text and target properties.
+ * @returns {*} Simple long form pattern
+ */
+const SimpleLongForm = ({ title, copy, variation, link }) =>
+  featureFlag(
+    SIMPLELONGFORM,
+    <section
+      data-autoid={`${prefix}--simplelongform`}
```

Comment:
Switch to the dds prefix.

Reply:
updated the code with [this commit](https://github.com/carbon-design-system/ibm-dotcom-library/pull/561/commits/db7c3e6a903c09f8496332f049b3ca59488ad70b)

Reply:
updated the code with [this commit](https://github.com/carbon-design-system/ibm-dotcom-library/pull/561/commits/db7c3e6a903c09f8496332f049b3ca59488ad70b)